supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -16,28 +16,51 @@ When the format _day Month year_ causes a presentation or clarity issue, use _Mo
 The Conscious Language Group supports the Red Hat commitment to remove problematic language from our code, documentation, websites, and open source projects with which Red Hat is involved.
 For more information about the Conscious Language Group, see https://github.com/conscious-lang/conscious-lang-docs.
 
-=== blacklist and whitelist
+You can use the following alternatives:
```

Comment:
This list doesn't make any sense without first listing the terms your suggesting alternatives *for*.

Reply:
oops my bad . So similar to what alternatives has been suggested for blacklist/whitelist. I have added these alternatives under master/slave to keep t consistent. what do you think?

Reply:
@stoobie

Reply:
@Amrita42 

The construction of the page still doesn't make sense to me when I scan it. And the suggested alternatives appear to be part of the second example in both headings. 

I again suggest:

When possible, rewrite documentation to avoid these terms:
- blacklist
- whitelist
- master
- slave
=== Blacklist and whitelist
<Suggest alternatives>
Example 1
Example 2
=== Master and slave
<Suggest alternatives>
Example 1
Example 2